index.bs

```diff
@@ -10798,13 +10794,25 @@ operation such as resolution of {{Promise}}s in the {{AudioWorkletGlobalScope}}.
 				[=Computing a block of audio|compute a block of audio=], and
 				[=Making a buffer available for reading|make it available for reading=].
 
-			4. If this {{AudioNode}} is an {{AudioWorkletNode}}, invoke associated
-				{{AudioWorkletProcessor}}'s {{AudioWorkletProcessor/process()}} method
-				to [=Computing a block of audio|compute a block of audio=] with
-				the argument of [=input buffer=], output buffer and
-				[=input AudioParam buffer=].
+			4. If this {{AudioNode}} is an {{AudioWorkletNode}}, execute these step:
+
+				1. Let <var>processor</var> be the associated {{AudioWorkletProcessor}}
+					instance.
+
+				2. Let <var>processFunction</var> be the result of
+					<code><a href="http://www.ecma-international.org/ecma-262/6.0/#sec-get-o-p">
+					Get</a>(O=<i><var>processor</var></i>, P="process")</code>.
+
+				3. If <code><a href="http://www.ecma-international.org/ecma-262/6.0/#sec-iscallable">
+					IsCallable</a>(argument=<i><var>processFunction</var></i>)</code> is
+					false, throw a {{TypeError}}.
```

Comment:
Not ReferenceError if the function doesn't exist?

Reply:
I am no expert on ES6 abstract operation, but technically `IsCallable` checks if the target is `function` type. So I think `TypeError` is correct here. 

Example: https://www.ecma-international.org/ecma-262/7.0/#sec-getmethod